src/wix/WixToolset.Core.WindowsInstaller/WindowsInstallerBackendErrors.cs

```diff
@@ -7,6 +7,11 @@ namespace WixToolset.Core.WindowsInstaller
 
     internal static class WindowsInstallerBackendErrors
     {
+        public static Message OpenDatabaseFailed(string databaseFile, string error)
+        {
+            return Message(null, Ids.OpenDatabaseFailed, "Failed to open database '{0}'. Ensure it is a valid database, and it is not open by another process. {1}", databaseFile, error);
```

Comment:
`...valid database, is writable, and is not open...`?

Reply:
Sure.